test/ServerTest.cpp

```diff
@@ -149,7 +149,9 @@ TEST(ServerTest, createResponseMetadata) {
   const Index& index = qec->getIndex();
   DeltaTriples deltaTriples{index};
   const std::string update = "INSERT DATA { <b> <c> <d> }";
-  ParsedQuery pq = SparqlParser::parseQuery(update);
+  auto pqs = SparqlParser::parseUpdate(update);
+  EXPECT_THAT(pqs, testing::SizeIs(1));
+  ParsedQuery pq = std::move(pqs[0]);
```

Comment:
Is that metadata also correct for chained updates, and shouldn't that case then also be tested?

Reply:
The metadata is independent of the chained updates. For chained updates an array of the json for the individual updates is returned. This is just for ensuring that we're indeed getting the expected number of parsed queries.